parent/pom.xml

```diff
@@ -576,6 +576,7 @@
     <protobuf-maven-plugin-version>0.5.1</protobuf-maven-plugin-version>
     <protonpack-version>1.8</protonpack-version>
     <pubnub-version>4.21.0</pubnub-version>
+    <pulsar-client-version>2.2.1</pulsar-client-version>
```

Comment:
the client property placeholder is present also at the line 760. Remove one of the two placeholder

Reply:
done